rollback_on_fail.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+rollback () {
+    wget https://github.com/Jellyfish-AI/jf_agent/archive/refs/tags/stable.tar.gz
+    tar -zxvf stable.tar.gz --directory ./
+    python jf_agent-stable/jf_agent/main.py
+}
+
+timeout --preserve-status 10h python jf_agent/main.py
```

Comment:
NIT: It'd be nice to have away around this hard 10 hour limit.... Generally the 'daily' ingest should never go beyond 10 hours, but what about initial ingest? I wouldn't be surprised if that could take over 24 hours for a large client

Reply:
Agreed, should have an override.